src/components/button/button.scss

```diff
@@ -6,15 +6,16 @@
 }
 
 a.bttn {
-   span {
-     margin-left: $sm;
-     color: $primary;
-   }
-   &.bttn--primary {
-     span {
-       color: white;
-     }
-   }
+  span {
+    margin-left: $sm;
+    color: $primary;
+  }
+
+  &.bttn--primary {
+    span {
+      color: white;
```

Comment:
Since this is just a spacing change, this comment is not totally relevant to this PR, but is this a default setting? It seems like we would need a bunch of modifications for background treatments immediately after this.